### ISSUE TITLE
Could not resolve all dependencies for configuration

### ISSUE THREAD

**Issue by HeinPauwelyn** [WatchFriends/Android#70]
Als ik project wil runnen, heb ik deze error:

```none
Error:FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugApkCopy'.
   > Could not find com.android.support:appcompat-v7:23.4.0.
     Required by:
         WatchFriends:app:unspecified
   > Could not find com.android.support:design:23.4.0.
     Required by:
         WatchFriends:app:unspecified
   > Could not find com.android.support:support-v4:23.4.0.
     Required by:
         WatchFriends:app:unspecified
   > Could not find com.android.databinding:library:1.2.1.
     Required by:
         WatchFriends:app:unspecified
   > Could not find com.android.databinding:adapters:1.2.1.
     Required by:
         WatchFriends:app:unspecified

* Try:
Run with --stacktrace option to get the stack trace. Run with --debug option to get more log 
output.
```

Weet er iemand hoe dit komt?

**Comment by HeinPauwelyn** [comment]
Gevonden: 

> This can happen when there is an error in an update. To fix it open android sdk manager and:

> * Uninstall android support repository
> * Install again android support repository

> <sup>Bron: [Stack Overflow](http://stackoverflow.com/questions/38090248/could-not-resolve-all-dependencies-for-configuration-appdebugapkcopy)</sup>

**Comment by omka243** [comment]
I also encountered this error.

This error caused by updating Android Support Repository.

Another way to solve this error is go to your build.gradle file and check for warnings related to support version dependencies. then look for hint, make change as it suggested. In my case this error caused by support dependencies:-

compile 'com.google.android.gms:play-services-maps:9.0.0'
and I changed to :-

compile 'com.google.android.gms:play-services-maps:10.0.0'
